StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetViewController.swift

```diff
@@ -619,14 +618,17 @@ extension PaymentSheetViewController: SavedPaymentOptionsViewControllerDelegate
         ) { (_) in
             // no-op
         }
+
         if !savedPaymentOptionsViewController.canEditPaymentMethods {
             savedPaymentOptionsViewController.isRemovingPaymentMethods = false
-            // calling updateUI() at this point causes an issue with the height of the add card vc
```

Comment:
This was part of the first commit that introduced PaymentSheet. I tested this codepath on a couple simulator versions and didn't see any problems, so I think it's safe to say this was since fixed.